war/src/main/js/add-item.js

```diff
@@ -107,7 +107,6 @@ $.when(getItems()).done(function(data) {
           btn.removeClass('disabled');
           btn.prop('disabled', false);
         }
-        btn.focus();
```

Comment:
Sure, I do not insist on my changes - just a suggestion. I think that check of name validity after each character is a useful feature. Since most of the issues with new job name check were solved recently, this is my only suggestion for the improvement left.